erpnext_com/www/reporting-security-vulnerabilities.md

```diff
@@ -0,0 +1,51 @@
+#Reporting Security Vulnerabilities
+<br>
+<p class='p' style='margin-bottom: 0;'>
+While we try to be proactive in preventing security problems, we do not assume they’ll never come up.
+</p>
+<p class='p' style='margin-bottom: 0;'>
+It is standard practice to responsibly and privately disclose a security problem to the vendor i.e. Frappe core development team before publicising, so a fix can be prepared, and damage from the vulnerability minimised.
+</p>
+
+# Policy
+<p class='p' style='margin-bottom: 0;'>
+You are responsible for complying with all applicable laws and must only ever use or otherwise access your own test accounts when researching vulnerabilities in any of our products or services. Access to, or modification of user data is explicitly prohibited without prior consent from the account owner.
+</p>
+
+# Qualifying Vulnerabilities
+
+<p class='p' style='margin-bottom: 0;'>
+Any reproducible vulnerability that affects the security of our users is likely to be in scope. 
+</p>
+
+Common examples include:
+
+* Cross Site Scripting (XSS)
+* Cross Site Request Forgery (CSRF)
+* Server Side Request Forgery (SSRF)
+* Remote Code Execution (RCE)
+* SQL Injection (SQLi)
+
+
+# Security Vulnerability Submission by Email
+<p class='p' style='margin-bottom: 0;'>If you found any security breaches, please report the issue <i class="fa fa-envelope"> </i> <a  href='mailto:report@erpnext.com'> Frappe Vulnerability Management</a></p>
+
+<p class='p' style='margin-bottom: 10px;'>
+It is important to include at least the following information in the email:
+</p>
+
+* Organization and contact name
+* Your Reference / Advisory Number
+* Description of the potential vulnerability
+* Supporting technical details (such as system configuration, traces, description of exploit/attack code, sample packet capture, proof of concept, steps to reproduce the issue)
+* Information about known exploits
+* Disclosure plans, if any
+* If you want public recognition
+
+# Considered Resources
```

Comment:
is this section required?

Reply:
To avoid custom apps related issues.

Reply:
How will some users know if its custom app or not, remove it. If they report, we can always say its not custom app.